### ISSUE TITLE
[TF2] [Feature Request] Replace the Sandman's negative stat.

### ISSUE THREAD

**Issue by Arbitz** [ValveSoftware/Source-1-Games#4330]
A small feature request. Changing the Sandman's downside from the punishing "-15 health on wearer" to "-20% melee damage". 

The -15 health is far too punishing for the small upside the weapon gives, leaving it pointless since the changes in 2017.

**Comment by FrozenDragon0** [comment]
I agree with this, considering how useless and buggy the slowdown effect actually is, the -15 health downside makes the weapon not even worth.

**Comment by jh34ghu43gu** [comment]
Upside in mvm due to slower overheal decay

Food for thought.

**Comment by Arbitz** [comment]
> Upside in mvm due to slower overheal decay
> 
> Food for thought.

Ok but MVM is just a side-mode to the main game, and balancing for the main game should take priority. And honestly the slower overheal decay isn't that important in MVM.
